### PR TITLE
fixed saved talent

### PR DESCRIPTION
**Pull request by Yulllia**
-fixed saved talent

### CODE REVIEW

File: src/modules/profile/profile.controller.ts [hunk 1]
```diff
@@ -88,13 +88,13 @@ export class ProfileController {
     };
   }
 
-  @Get(':id')
-  getProfileSettings(@Param('id') id: number) {
-    return this.profileService.getProfileSettings(Number(id));
+  @Get(':id/:clientId')
+  getProfileSettings(@Param('id') id: number, @Param('clientId') clientId: number) {
```
Comment (lyudad) [inline src/modules/profile/profile.controller.ts]: return type

File: src/modules/profile/profile.controller.ts [hunk 2]
```diff
@@ -88,13 +88,13 @@ export class ProfileController {
...
+    return this.profileService.getProfileSettings(Number(id), Number(clientId));
   }
 
   @Put(':id')
-  updateSingleProfile(@Param('id') id: number, @Body() saved: { saved: boolean }) {
+  updateSingleProfile(@Param('id') id: number, @Body() saved: { saved: boolean; clientId: number }) {
```
Comment (lyudad) [inline src/modules/profile/profile.controller.ts]: everywhere return type

File: src/modules/profile/profile.service.ts [hunk 3]
```diff
@@ -35,19 +39,32 @@ export class ProfileService {
     return allProfile;
   }
 
-  async updateSingleProfile(id: number, save: { saved: boolean }) {
-    const { saved } = save;
-    if (id && save) {
-      await this.profileRepository.update({ id: id }, { saved: saved });
+  async updateSingleProfile(id: number, save: StatusDto) {
+    const { saved, clientId } = save;
+    const profile = await this.statusRepository
```
Comment (lyudad) [inline src/modules/profile/profile.service.ts]: why it called statusRepository? if it represent savedUsers?

Comment (Yulllia) [reply src/modules/profile/profile.service.ts]: yes it represents saved profile freelancer

File: src/modules/profile/profile.controller.ts (hunk 2 above)
Comment (Yulllia) [reply src/modules/profile/profile.controller.ts]: fixed

File: src/modules/profile/profile.controller.ts (hunk 1 above)
Comment (Yulllia) [reply src/modules/profile/profile.controller.ts]: fixed

File: src/modules/profile/profile.service.ts (hunk 3 above)
Comment (lyudad) [reply src/modules/profile/profile.service.ts]: why it status, I dont like this name, it not give me information what is represent. code should be self documented

Comment (Yulllia) [reply src/modules/profile/profile.service.ts]: fixed

File: src/modules/profile/profile.service.ts [hunk 4]
```diff
@@ -35,19 +39,35 @@ export class ProfileService {
...
   }
 
-  async updateSingleProfile(id: number, save: { saved: boolean }) {
-    const { saved } = save;
-    if (id && save) {
-      await this.profileRepository.update({ id: id }, { saved: saved });
+  async updateSingleProfile(id: number, save: SavedProfileDto): Promise<StatusEntity[] | SavedProfileDto> {
+    const { saved, clientId } = save;
+    const profile = await this.saveProfileFreelancerRepository
+      .createQueryBuilder('StatusUpdating')
```
Comment (lyudad) [inline src/modules/profile/profile.service.ts]: why Status updating?

Comment (Yulllia) [reply src/modules/profile/profile.service.ts]: I rename it to saveProfileFreelancer

File: src/modules/profile/profile.service.ts [hunk 5]
```diff
@@ -35,19 +39,35 @@ export class ProfileService {
...
   }
 
-  async updateSingleProfile(id: number, save: { saved: boolean }) {
-    const { saved } = save;
-    if (id && save) {
-      await this.profileRepository.update({ id: id }, { saved: saved });
+  async updateSingleProfile(id: number, save: SavedProfileDto): Promise<SaveFreelancerEntity[] | SavedProfileDto> {
+    const { saved, clientId } = save;
+    const profile = await this.saveProfileFreelancerRepository
+      .createQueryBuilder('Freelancer')
```
Comment (lyudad) [inline src/modules/profile/profile.service.ts]: please use lowercase instead of Upper case for each QueryBuilder

Comment (Yulllia) [reply src/modules/profile/profile.service.ts]: fixed it

**Pull request merged**